[error] java.io.FileNotFoundException: 
[error] Could not automatically find the JDK/lib/tools.jar.
[error] You must explicitly set JDK_HOME or JAVA_HOME.
[error]     
[error] 	at org.scaladebugger.SbtJdiTools$.$anonfun$JavaTools$6(SbtJdiTools.scala:52)
[error] 	at scala.Option.getOrElse(Option.scala:189)
[error] 	at org.scaladebugger.SbtJdiTools$.JavaTools$lzycompute(SbtJdiTools.scala:52)
[error] 	at org.scaladebugger.SbtJdiTools$.JavaTools(SbtJdiTools.scala:40)
[error] 	at org.scaladebugger.SbtJdiTools$.$anonfun$baseScalaDebuggerToolsSettings$1(SbtJdiTools.scala:29)
[error] 	at sbt.std.Transform$$anon$3.$anonfun$apply$2(Transform.scala:47)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:69)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:283)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:24)
[error] 	at sbt.Execute.work(Execute.scala:292)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:283)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:65)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:750)
[error] (Compile / unmanagedJars) java.io.FileNotFoundException: 
[error] Could not automatically find the JDK/lib/tools.jar.
[error] You must explicitly set JDK_HOME or JAVA_HOME.
[error]     
